services/app-web/src/pages/StateSubmission/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -156,14 +204,58 @@ export const ReviewSubmit = ({
             <Modal
                 modalRef={modalRef}
                 id="review-and-submit"
-                modalHeading="Ready to submit?"
+                modalHeading={unlocked ? 'Summarize changes' : 'Ready to submit?'}
                 submitButtonProps={{ className: styles.submitButton }}
-                onSubmit={handleFormSubmit}
+                onSubmitText={unlocked ? 'Resubmit' : undefined}
+                onSubmit={() => {
+                    setFocusErrorsInModal(true)
+                    formik.handleSubmit()
```

Comment:
I wonder if we can just call the `onSubmit` directly here when we are in a draft submission rather than going through formik even though we don't have a form in that modal.

Reply:
Another approach I was thinking about doing was to check for unlocked right after `setFocusErrorsInModal(true)` then calling the appropriate function
```
    const submitHandler = async() => {
        setFocusErrorsInModal(true)
        if (unlocked) {
            formik.handleSubmit()
        } else {
            await onSubmit(undefined)
        }
    }
```  

```
      onSubmitText={unlocked ? 'Resubmit' : undefined}
      onSubmit={submitHandler}
```